ui/lib/data-fetch.js

```diff
@@ -28,11 +28,11 @@ const dataFetch = (url, options = {}, successFn, errorFn) => {
 
         return result;
       } else {
-        throw res
+        throw res.text()
       }
 
     }).then(successFn)
-    .catch((res) => res.text().then(errorFn));
+    .catch((e) => e.then(text => errorFn(text)));
```

Comment:
@Abhishek-kumar09 Here one integration test is failing `errorFn` Any suggestion?

Reply:
@Abhishek-kumar09 any quick suggestions to clear this check?